### DISCUSSION TITLE
Get Serial Cmd ET when Meter Device is DUMMY with Symbolic BT from Mastech 6514 Extra Device

### DISCUSSION THREAD

**Discussion by rare-bean** [artisan-roaster-scope/artisan#1636] [Q&A] [answered]
Hi.

I have a device controlled over USB serial that sets fan and heater power on an SR800. This is configured as a DUMMY type meter in the ET/BT tab of device configuration. It has been tested successfully with sliders, buttons, and software PID control. 

BT temperature is read from a Mastech 6514 setup as the second device in the Extra Devices tab, with "Y6" entered in the BT Y(x) in the Symb ET/BT tab. Displayed BT value in Artisan matches the physical meter's LCD display.

The USB SR800 controller can also read the SR800 NTC resistance and I would like to use this for the ET value in Artisan. How do I set Artisan to send a serial command to the DUMMY device and use the returned integer value as an input data channel? For example, if DUMMY is set as first Extra Device, I would use Y3 or Y4 in the Symb ET Y(x) to link the returned serial data. 

Thanks!

**Comment by MAKOMO** [comment, marked as answer]
The DUMMY device is like its name suggests a dummy device not reading data from anywhere. So reading from a serial source with this device is not possible. You need to develop a real device type in Artisan for the SR800 which does both, fetching reading and sending control commands.

**Reply by rare-bean** [reply]
Okay. Thanks for clearing this up and I will look into developing device types.

This version of the USB controller is already usable as intended, since the SR800 NTC isn't a great indicator of BT. The DUMMY device worked very well for testing the USB controller quickly.
